lib/core/config.js

```diff
@@ -148,6 +148,7 @@ Config.prototype._mergeConfig = function(configFilePath, defaultConfig, env, ena
 
   let config;
   if (fs.existsSync(configFilePath + '.js')) {
+    delete require.cache[fs.dappPath(configFilePath + '.js')];
     config = require(fs.dappPath(configFilePath + '.js'));
   } else {
     config = fs.readJSONSync(configFilePath + '.json');
```

Comment:
This could be moved to the previous `if (!fs.existsSync(configFilePath + '.js') && !fs.existsSync(configFilePath + '.json'))` on line 141.

Reply:
I agree, but different PR (you're saying if JS, if json, else file not found, make sense)